src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -335,12 +335,16 @@ namespace OpenRCT2::Platform
 
     std::vector<std::string_view> GetSearchablePathsRCT1()
     {
-        return {};
+        return {
+            R"(/usr/share/games/roller-coaster-tycoon)",
```

Comment:
Could you add a comment to both of these that these are used by game-data-packager?